tests/unit/controlgroup/controlgroup.html

```diff
@@ -58,6 +58,16 @@
 		<label class="ui-controlgroup-label">Label</label>
 		<button>Button with icon on the bottom</button>
 	</div>
+	<div class="controlgroup-single-select">
+		<select id="select-single">
+			<option>Fast</option>
+			<option>Medium</option>
+			<option>Slow</option>
+		</select>
+	</div>
+	<div class="controlgroup-single-simple">
```

Comment:
Let's rename this to `controlgroup-single-button` to match the one above (also update the test titles).